reana-ui/src/pages/workflowDetails/WorkflowDetails.js

```diff
@@ -2,7 +2,7 @@
   -*- coding: utf-8 -*-
 
   This file is part of REANA.
-  Copyright (C) 2020, 2022, 2023 CERN.
+  Copyright (C) 2020, 2022, 2023, 2024 CERN.
```

Comment:
A functional observation. Whilst the workflow is running, on the detailed record page, a step may be "completed", but the UI still says "created" next to the step selection drop-down:

![created](https://github.com/reanahub/reana-ui/assets/517546/dbfccc94-ec3f-48f6-9fbf-8233f01d1452)

Reply:
Probably unrelated to this PR, let's check and fix this as part of another issue/PR?